float64[] delta_pose_rpy
